### PR TITLE
Rk/add footer

### PR DESCRIPTION
**Pull request by rkaufman13**
Small one-- adds a footer to every page and an about page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -41,6 +42,11 @@ function App() {
           </Row>
         </Container>
       </main>
+      <Container className="footer">
```
Comment (eggchan) [inline src/App.js:45]: This should live inside of a `<footer>` tag. Ideally remove the footer class. And there needs to be a `<Col xl={8}` inside the row and then call the Footer

File: src/About.jsx
Comment (eggchan) [inline src/About.jsx:1]: A suggestion of how this should be laid out:
```
<Container>
 <Row>
  <Col>
   <h2>About Noodle</h2>
    <p>Scheduling events <em>used</em> to be easy on the other sites. Then they started to suck. Intrusive ads, pop-ups, forced logins, slow-loading pages, all that jazz.</p>
    <p>Noodle is a new take on event scheduling. No ads, no logins, no emails you don't want.</p>
    <p>Noodle was made by <a href="https://github.com/rkaufman13">Rachel Kaufman</a>, with lots of help from <a href="#">Sarah Tonsager</a> and <a href="https://chriscombs.net">Chris Combs</a>.</p>
   </Col>
  </Row>
</Container>
```

File: src/Footer.jsx
Comment (eggchan) [inline src/Footer.jsx:1]: We should consider how this is going to look on a mobile device, e.g. it should stack once it gets to Bootstrap size sm.

**Review by eggchan — CHANGES_REQUESTED** [review]
A few suggestions and a few syntax fix requests

File: src/Footer.jsx
Comment (rkaufman13) [reply src/Footer.jsx:1]: it actually looks OK on the device simulator...the margins of the main container, on the other hand, are not great, but that's a different issue.
Screen Shot 2023-07-27 at 8 11 59 PM

Comment (eggchan) [reply src/Footer.jsx:1]: imo it should still stack but if you don't want it to I'm fine with leaving as is 🙂

**Review by eggchan — APPROVED** [review]

**Pull request merged**